lib/puppet/application/ssl.rb

```diff
@@ -99,35 +113,51 @@ def verify(host)
 
     key = host.key
     unless key
-      puts "The host's private key is missing"
+      puts _("The host's private key is missing")
       exit(1)
     end
 
     cert = host.check_for_certificate_on_disk(host.name)
     unless cert
-      puts "The host's certificate is missing"
+      puts _("The host's certificate is missing")
       exit(1)
     end
 
     if cert.content.public_key.to_pem != key.content.public_key.to_pem
-      puts "The host's key does not match the certificate"
+      puts _("The host's key does not match the certificate")
       exit(1)
     end
 
     store = host.ssl_store
     unless store.verify(cert.content)
-      puts "Failed to verify certificate '#{host.name}': #{store.error_string} (#{store.error})"
+      puts _("Failed to verify certificate '%{name}': %{message} (%{error})") % {
+        name: host.name, message: store.error_string, error: store.error
+      }
       exit(1)
     end
 
-    puts "Verified certificate '#{host.name}'"
+    puts _("Verified certificate '%{name}'") % {
+      name: host.name
+    }
     # store.chain.reverse.each_with_index do |issuer, i|
     #   indent = "  " * (i+1)
     #   puts "#{indent}#{issuer.subject.to_s}"
     # end
     exit(0)
   rescue => e
-    puts "Verify failed: #{e.message}"
+    puts _("Verify failed: %{message}") % { message: e.message }
     exit(1)
   end
+
+  def clean(host)
+    settings = %w[hostprivkey hostpubkey hostcsr hostcert passfile]
+    settings.concat(%w[localcacert hostcrl]) if options[:local_ca]
+    settings.each do |setting|
+      path = Puppet[setting]
+      if Puppet::FileSystem.exist?(path)
+        Puppet::FileSystem.unlink(path)
+        puts _("Deleted %{path}") % { path: path }
```

Comment:
Nit, I think elsewhere in puppet (and in `puppet cert clean`) we say "Removed ...". What does `puppetserver-ca` do?

Reply:
We're saying "Cleaned files for..." in the CA CLI, as a collective, since the cleaning is done via an API call.